src/test/java/reposense/git/GitRemoteTest.java

```diff
@@ -0,0 +1,21 @@
+package reposense.git;
+
+import java.util.Map;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import reposense.template.GitTestTemplate;
+
+class GitRemoteTest extends GitTestTemplate {
+
+    @Test
+    void getRemotesTest() {
```

Comment:
The name of the function should follow something like the format `featureUnderTest_testScenario_expectedBehavior()`. I think if possible, it would be nice to test `getRemotes()` and `getAvailableRemoteLocation()` separately.